app/javascript/components/CollabsPage.tsx

```diff
@@ -66,31 +67,32 @@ const CollabsPage = ({
                 learn more to get started
               </a>
             </p>
-          </Placeholder>
+          </
+            >
```

Comment:
It looks like you didn't test again after doing your modifications. Typescript raises an error here.
Please fix this and update your screenshots.